### ISSUE TITLE
The title of the article isn't specific enough (hard to find with Google/Bing)

### ISSUE THREAD

**Issue by AngelosP** [aspnet/Docs#9181]
This article is explaining how to troubleshoot startup errors in Azure App Service. This is good content, but the title doesn't accurately reflect the contents. As a result it's really hard to find this article through a Google/Bing search or even through a TOC search. I suggest the title is renamed to "Troubleshoot ASP.NET Core startup errors on Azure App Service"

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 587ea934-c90e-54a3-c708-83d36cf621b4
* Version Independent ID: c2f26258-1119-e6c2-3f2d-d0f0cddbdb7f
* Content: [Troubleshoot ASP.NET Core on Azure App Service](https://docs.microsoft.com/en-us/aspnet/core/host-and-deploy/azure-apps/troubleshoot?view=aspnetcore-2.1)
* Content Source: [aspnetcore/host-and-deploy/azure-apps/troubleshoot.md](https://github.com/aspnet/Docs/blob/master/aspnetcore/host-and-deploy/azure-apps/troubleshoot.md)
* Product: **aspnet-core**
* GitHub Login: @guardrex
* Microsoft Alias: **riande**

**Comment by guardrex** [comment]
Seems like a good idea to me.

@Rick-Anderson @scottaddie @tdykstra Thoughts?

**Comment by guardrex** [comment]
Yes, sounds good ... just want to make sure we intend to continue to point out to the Azure docs for post-startup monitoring and troubleshooting.

When I make this update (after we've collected more team feedback), I'll leave the TOC alone.
